src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findTabById } from '../../utils';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab) => {
+          const currentTab = findTabById(tabs, selectedTabId);
```

Comment:
Try moving this variable above and use also for the content value because you are repeating the same function twice